docusaurus.config.js

```diff
@@ -72,6 +73,7 @@ const config = {
           editUrl: 'https://github.com/MetaMask/metamask-docs/edit/main/',
           sidebarPath: false,
           breadcrumbs: false,
+          remarkPlugins: [npm2yarnPlugin],
```

Comment:
Just a question, do we require to add it in every instance?

Reply:
It currently looks like remarkPlugins cannot be declared globally for our site. Looks like there's an open issue to address this: `https://github.com/facebook/docusaurus/issues/5999`

Reply:
I see, thanks!